Source/Core/Core/WiiUtils.cpp

```diff
@@ -461,10 +461,11 @@ OnlineSystemUpdater::Response OnlineSystemUpdater::GetSystemTitles()
   std::string base_url = Config::Get(Config::MAIN_WII_NUS_SHOP_URL);
   if (base_url.empty())
   {
-    // Note: We don't use HTTPS because that would require the user to have
-    // a device certificate which cannot be redistributed with Dolphin.
-    // This is fine, because IOS has signature checks.
-    base_url = "http://nus.shop.wii.com";
+    // The NUS servers for the Wii are currently offline as of PR 10634, but the backing CDN is
```

Comment:
I think referring to https://bugs.dolphin-emu.org/issues/12865 is better than the PR. It contains a more complete timeline and explanation of the problem. (Though you can always refer to the PR too if you think that's a good idea)